plugins/mentions/mentions.py

```diff
@@ -119,17 +141,31 @@ async def handle_message(self, evt: MessageEvent) -> None:
                 slack_bridge_ignore = True
 
         # collect group names & users to notify
-        users_to_notify, group_names = self.get_info_on_matches(matches)
+        users_to_notify, group_names, quote_triggering_message, always_reply_in_thread = self.get_info_on_matches(
+            matches
+        )
 
         # construct body and formatted_body for message
         body = formatted_body = "Pinging members of " + ', '.join(group_names) + ": "
         body += ", ".join(users_to_notify)
         for user in users_to_notify:
             formatted_body = formatted_body + " <a href='https://matrix.to/#/" + user + "'>" + user + "</a>"
 
-        await self.notify_users(users_to_notify, body, formatted_body, slack_bridge_ignore, evt)
+        await self.notify_users(
+            users_to_notify,
+            body,
+            formatted_body,
+            {
```

Comment:
Just a detail, but I find it a bit weird that some of the arguments are passed directly, and others are passed as an entry in a dict.

Reply:
Not different from defining the last argument as a dict and then passed that variable, right?

Reply:
done!